packages/prosess-aarskvantum-oms/src/components/utils.ts

```diff
@@ -11,4 +11,18 @@ export const durationTilTimerMed7ogEnHalvTimesDagsbasis = (delvisFravær: string
   return heleDager * 7.5 + resttimer;
 };
 
+const koronaStartdatoFom = moment('2020-03-13', ISO_DATE_FORMAT);
+const koronaSluttdatoTom = moment('2020-12-31', ISO_DATE_FORMAT);
+
+export const periodeErIKoronaperioden = (periode: string): boolean => {
+  const [fom, tom] = periode.split('/');
+  const datoErIPerioden = dato =>
+    moment(dato, ISO_DATE_FORMAT).isBetween(
+      koronaStartdatoFom.subtract(1, 'second'),
+      koronaSluttdatoTom.add(1, 'second'),
+    );
+
+  return datoErIPerioden(fom) || datoErIPerioden(tom);
```

Comment:
det holder at bare deler av perioden er innenfor koronaperioden?

Reply:
Ja egentlig. Men det er kanskje unødvendig, siden backend uansett vil splitte opp perioder som krysser start- og sluttidspunkt for korona